x-pack/plugin/watcher/src/internalClusterTest/java/org/elasticsearch/xpack/watcher/actions/webhook/WebhookTokenIntegrationTests.java

```diff
@@ -85,14 +85,18 @@ public void stopWebservice() throws Exception {
         webServer.close();
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/93536")
     public void testWebhook() throws Exception {
+        assumeFalse(
+            "Cannot run in FIPS mode since the keystore will be password protected and sending a password in the reload"
+                + "settings api call, require TLS to be configured for the transport layer",
+            inFipsJvm()
+        );
         String localServer = "localhost:" + webServer.getPort();
         logger.info("--> updating keystore token hosts to: {}", localServer);
         Path configPath = internalCluster().configPaths().stream().findFirst().orElseThrow();
-        try (KeyStoreWrapper ksw = KeyStoreWrapper.bootstrap(configPath, () -> new SecureString("".toCharArray()))) {
+        try (KeyStoreWrapper ksw = KeyStoreWrapper.create()) {
```

Comment:
This change is not strictly necessary for the fix. But using `KeyStoreWrapper.bootstrap` seems a bit odd to me: The keystore does not exist. So it will be created. When creating a new keystore, the given password is completely ignored and the keystore is always created with an empty password. In this case, it seems to me that using `create` is better for clarity. 

I am also curious on why `bootstrap` ignores the password when creating a new keystore. But that's a separate issue.